server/gokbg3/src/main/webapp/setup/DDC_english.txt

```diff
@@ -0,0 +1,1000 @@
+000;Computer science, information & general works;
```

Comment:
is it wanted that the ending is txt and not csv?

Reply:
Yes